lib/sync.c

```diff
@@ -1868,6 +1868,9 @@ iscsi_discovery_cb(struct iscsi_context *iscsi, int status,
                 struct iscsi_discovery_address *datmp;
 
                 datmp = malloc(sizeof(struct iscsi_discovery_address));
+		if (datmp == NULL) {
+			return;
+		}
```

Comment:
Please follow the existing code style and use spaces to indent code instead of tabs.